cmd/resourcegroup.go

```diff
@@ -18,7 +18,7 @@ package cmd
 import (
 	"fmt"
 
-	"github.com/github.com/RithvickAR/aksctl/coreaksctl"
+	"github.com/RithvickAR/aksctl/coreaksctl"
```

Comment:
This should be : 	"github.com/RithvickAR/aksctl/coreaksctl"
	"github.com/adfolks/aksctl/coreaksctl"
Right? Please change all refernces

Reply:
changed